server/server.js

```diff
@@ -150,68 +150,85 @@ app.route('/login')
         }
     });
 
-app.get('/dashboard/:installationId?', async (req, res) => {
-
+app.get("/dashboard", async (req, res) => {
     if (!req.session.user || !req.cookies.user_sid) {
-        res.redirect('/login')
+        res.redirect("/login")
         return
     }
-    else if(typeof req.params.installationId !== "string") {
+      
+    try {
+        let allInstallations = await db.getInstallations()
+            
+        let viewParams = {
+            installations: allInstallations
+                .filter((installation) => installation.isActive)
+                .map((installation) => {
+                    return { name: installation.name, id: installation.id }
+                }),
+        };
+        viewParams.viewMessage = allInstallations.length >= 1 ? "Please select an installation" : "No installations to display"
```

Comment:
Can you please change this to use single quotes?
There are a few other places too, could you just double-check that all your new changes use singles instead of doubles? Thanks!